code/procs/scanprocs.dm

```diff
@@ -198,11 +198,9 @@
 
 
 	var/datum/statusEffect/simpledot/radiation/R = M.hasStatus("radiation")
-	var/datum/statusEffect/simpledot/radiation/NR = M.hasStatus("n_radiation")
-	if (R)
-		rad_data = "&emsp;<span class='alert'>Radiation poisoning: Lv [R.stage]</span>"
-	if (NR)
-		nrad_data = "&emsp;<span class='alert'>Neutron Radiation poisoning: Lv [NR.stage]</span>"
+	if (R?.stage)
+		rad_data = "&emsp;<span class='alert'>Radiation poisoning: Lv [R.stage] Dose: [M.radiation_dose] Sv</span>"
```

Comment:
probably don't need to include stage, since that only affects stun/mutation chances

Reply:
gonna make this print the same stuff as mousing over the status effect. I want docs to be able to figure out how fucked the patient is without necessarily needing to have an intuitive grasp of Sieverts.